### ISSUE TITLE
[Sybil Report]

### ISSUE THREAD

**Issue by mejiwef** [LayerZero-Labs/sybil-report#7867]
---
name: Sybil Report
about: Report Sybil Activity on LayerZero
title: "[Sybil Report]"
labels: under review
assignees: LayerZero-GH

---

# Reported Addresses
```bash
0xe293e7e9ed066c5869158c0c6827dc24d287e5cf
0xea1cf2b521e77017b1d945e06ff4f5b1363cdd23
0x73bd4f8768e110466b1b2f57c7b9c688ab46c649
0x21b1de665d73327e2841997e7ca1c2c3b79e08f8
0x169ec4f9525bd7d377b0d7709f4b3ac13f18e80a
0x465d69f6985a3da746bb31ea00e0ddba3e740e07
0x7c219f42a91ca2a79baa76cdd86fbd682f92df10
0x8789d8839f96540b1c1490a07ccfe596228c0f4f
0x9111286ede497648823a39545a1bcf106cdc166d
0x420ad85f3c32f28d93e0c47211f55d2e69fa8483
0x06a9a9d32ff61935f4cc094304d882e0ea3b7e36
0xeb395c5810a245c95c1a43cdbac3da8acf38da90
0x5558336a3ddb0aa015ef7f55cdbd2d9d40a6e955
0x3344ccf69a8727fa0e23451836a86d2aea260501
0xa148840d100c7a445c3609b71231625d6451a552
0x4923d9753d807b4ecb37e6e3fd3ee65981bb2043
0xe3cf0846597b12d3301360174fca7995f71d76fe
0x3f748f1ceee3e8e10166894149fe3ce238902ab4
0x14da2d182cfbc8bfbd1f345d220f4e34479748a1
0x9ad2ef62f1c08e2dafbfdf30384c9fb2e08fcd64
0x4dbb6d302e248ab855bf3feb6a5e7d6bf436e045
0x8b0a2957ad6e90ec4b7376870de5cfed873201cf
0x4f0ec2a6d1eb43552d7ad9ef1e125a7917253372
0x40e7b305a0ea54b4ec152b8bba4d80042e71028f
0xc094120e45a96d6b280ecbab341031a834d551c7
0x65915d7fa75021db9c9525787da82227d1fb9aed
0x8c242afb1b4d4074134a7b955ca4e7c82f6b740c
0x16967e0740d93f6d632215ad2141555276027c8a
0x1df56cb3396c428240bfc3633e0d81bd09f91269
0xc1b5d387d50912e4d40e342d4a7401de3ba539f5
0xcbab3e6af59387147c4ae6b9a1cb9034d53b2bef
0x648d1702cbdc9cdf1c86592149283a12afb3efd9
0x3583d2b2cbc98de8cceb51557611bf2d18140260
0x1a740c76c401f96aa4a6c9a82a084583a6cf06cf
0xbd2644ac1f2a91e979d995c807f5272a4db153aa
0xa8578dad9d8d8e259039f0edd8d15015227c54b1
0x5d1d2b0d0c56b2c0cf64a4850d35324bf01b85fe
0xd4e09e53aaf244658378e74d8c08421986daac6f
0x7c086f19f82df7f1a26d732b24a247b7f6fbfa9c
0xdbb912f7e6697ceb693f5cc230b2c3fad17b9d42
0x4e534780976ab23554443c3979af5927d5f0820c
0x32656e2fb2ce2f9785bdc40eba34d7b105512cb6
0xf143640d00ae8301600e277ad574760fd9249110
0xfd8d2d7c801c3541c1971086852f06df917c671a
0xd825bfded1301569513d9ad0187f627292cb88e0
0xaae195a440212ef2852953ada0f52592fa972e8a
0x98797c38224b50683e2a3d56a0ac362d06a160f2
0x621efc09cb4119f63a8495c41f5464df0366cd92
0x96f456bfb95551b7e7ad23827fddac661a85585d
0xdbd3905a239feaa6004c8313e38f122597e86f19
0x606cb301ebaec389224b43e18233305c15900d53
0x4ae41f470cd23a02c823048fd01508cf5478b2af
0xc4229cd2dee1ee32b0e8a4187aed19c26f43a85b
0xa841016a30d61286bfe24d12ba57ab25437c4855
0xf61e01033889bcb077133c0605e3d55861e03d1b
0x7add574176f930844c0518d910ebeeee3c74a771
0x2c1112a3b6e3547910cb76d82d893c066ae057d1
0xc76d7e641415812c15919d965a0b93f56069d644
0x4265bf3080fe913e3cb6d97c6f354943f6be23a0
0x9fe10a8e113c5c490eb5daeb144f8862354c8674
0x41c676e1ccbe4964de659496fa524b597f4ec1bf
0x12c2b37d74b3aa28efa0e677439e5d08d2d51258
0x8e4bff91fcfb256a9c5db2d46396b66c4b8ccac9
0x989713f5180bd3329480a165550c0028da503e13
0x3ff7fb4718ee653354855f8cab91ea72024ad0bf
0xd76c1a6bc8bf352fbe2daac61c0917e21ad55ac7
0x630d1277ded28099a5f325c173681a18c7c6829d
0xf416f86d931327df30c42d30caf74b8a53ffc659
0x214d9000c10850b786c00c75a29887fcd803d837
0x659273c6e0486bfe9d1c273601bdb584f69c812e
0x79cda6d47e94e87b519190de974f0b0137d1fc50
0x1566fd9fb171865b1466c8607acd1a4e7f726107
0xfe2eb4d560b6ca6819381ac6521a81e2ed936d73
0x20330c6da262dcd3c1529b0c82c2907de9319861
0xd4880602bbc640299bde68c3fd40897173b74352
0xa9d078ae88de536fd1c7d570d9f96e5c7122f5a4
0xfb9aec51890f6b79f0c33f79ba384a15971892c4
0xd7b89a45a72d66abdf38d2845ecf7ed84aa7f6e3
0xaa5a594b2c9f420cca9837fb486cb9264af1ff27
0xc5e9d5be9cdccf5bdac37b49427f8d1820f897fb
0x3ee749d5f1b0ff2bcabc903859f509e214caeab9
0x647763aa0db5c96c76f242692e3b7354732dac57
0x0ba2aa6c94a69be019f28786134cb1c8be033697
0xb6cf9da0d10de821c0c2a98dffd3bd365877c913
0xf7c722a74856af61809f9cddb90dc4f34cf221e1
0xcdc843c6808a949cfc675590f8abc5baad717558
0x3b94fbaf832237aa128b35d330399bc6e4cc6dc8
0x86466273b62e53ad88289c16c59b844a09ce2e39
0x3a1c7666f178c41c0d3604dd1de2b3b54982196e
0xdcb7b99d93af073b223bd2b1483e7e1d201fd69e
0x67e4c1094f96e5a339148536c6938ddc16e00fef
0xa495c34ada61a1d3a3ca88968d0756a563715ff0
0xe1d92d30a76b963111a9c02b9c484b3794f94154
0x5e2937355fc496c8a3ce506dd5be3fe5b47601b6
0xb451989710d474e7818973036ada7913370d3ce2
0x1bbe8e8b4572fb116b96e0be389a1bae7061b1e8
0xd0be7cee4968792ad1201bfefaa8f5f282966a5e
0x43efba616e14977b6b15e9e73a62e972ba64f079
0x95d7b5642e80a44975cb98d2767f14c7c3b65398
0x1e5761dd248277d092d0ac1f19ed4bc64bb6e4c5
0x291f0a8f5372f368f66d5eda0fcf3c5dbb799eb4
0x7d5c41afc92ce65dca85ac9b8c7fcf13c68435c0
0x1cef89bc94a63510348e9218568dcded43b8ddb2
0x0ddc82c1b7b1d7487af082483435b400a0377631
0x51fc925434ad50e1f75f53c6c7803cb6fba1c0e2
0x1b0ddf27c291b91949f267790c5a7013ae86ec8a
0xd0f2a4e31a1943a1770e2f8df3cdb2f9041ad55c
0x5cb43f32f3ef7d695493ca917e6d034d22b7336f
0x746eaf01d1023cb53812797d7d9d8173471a88e4
0x033bbd2dcc9fa6b73728ca6808551873da283c2a
0x0befa07bd91a5c1bf670d3113698440ee6678fd3
0x933314e394a383740e84e3761c8834f09876b5d1
0xf890971e114c01ec98ab3682148555409aec6f1c
0x8a721dd76a8b4dcb8a511c76206256f14da011a6
0xd1309b902824f0759b1c37cde9fa19cd9354b50a
0x6a2158fca057d3dab3063fbbcbe5f13b186ccf2e
0x4476cfe91d2e5756c9f73302f1e14cd5ea97be33
0x31cb9c3c950740a24f9093cf008ba6505840285e
0xa6a5d8481485d8124869cfd6042c813d73962e12
0xb97fd95ec730eae650fe722c08a1c6a899e97e79
0x6462de89bac2c79e20536d6c80e75730d09674d4
0x5c514da8626717a738dd8e9214043077f322f0d2
0x3612767b3870a2b68048a30d55dee67f17f7ca3a
0x0c79c7f173dcc5f686433000b0a584ab7c556daa
0xb6c7e4bd5ecfba2c83c0c335e778080c4ece5a8f
0xb54a4e350993c4c709b2bf4e9ccbecbdd8440bc8
0x92d98c7edb6339eaa07f7cccce1325bb1a1bacd9
0x3d1d68699a94dd0b892239cb4099e61a9b109e15
0xdbfcd4e0f28a3660f1a1a5dc65f6140432fd4059
0xdabad6fd361050fd6c051243178dab207a328fea
0x7456caca218b027760eebcd0e9282cff0cadb767
0x4c6ff3283028f874153c155490c3338cee11371e
0x3b4cc650435b6e423810c12a612f5f4e6f35921b
0xb3dbf65659c8a547ba2ed2b4c05a10aad9d9e526
0xcf3cb00f0352af83217d9d6af8a93ddaac09c8f4
0xbfce1dc750b390ca4e45e6d9283b30ccdc1d2ec3
0xe7bbfbfddc4c4c402fc1244eaae153f6c5be38b0
0xfe6844f8d0c109801219ad3ada08181fd6decb71
0xf71af79385fd8952ed89edddb6dab1839bfd400b
0x5c39588c5b8982e550e9fca0dc62fa37cd068c09
0xdcab05431675c05e9e8d6f73a8117028019f3dcc
0x7c711c4f23b29bfbe6acd634f218debd38a45904
0x8bbad5fa2e54d150bbc8552896fc8f8c334029ca
0xe193b65657d81e99df0f6007eea99cbe07e2c046
0xf500221972098c65cd289655d671472f61dfcb9b
0x38dba427a575ffdb824ff1686e0fb69b07dba553
0xcba5f2a8118ed20860ba086e1a2c0d8fe56834e6
0x0db765a37b505c70615fb6064a6cf2dfab39e474
0x9779696a406e88ca6f940654fcf8106da297110d
0x8a36a11e7d153e192cdf2f3f4e6b9b6c4e182577
0x1afc219fbc986e1610b3ff48fc160faaebf27b30
0xb9b5883aef6d055f90709a8a293192b58a388f01
0xe8b46bf604c3828ad48b5513e5c5041a186c1617
0x4264d1287e5140be35f0f03e169f4ee51fcdf9cd
0xb1cf854371923c46c70092d9534476b5f7ec6bc8
0x366db3cf21520f769e2762c85aa17a43f59bec81
0x34d17a4316ca6ad2cd4150d92406d7cc95be6014
0x014aa9da660b78dc74f17131e7be5fd8c753e660
0xc3d86b09c59fec287e3a11cf5122048f09b6d745
0x760d7e216495f8e64e282f42988d02e15806134d
0xab4d8e1c4ebbb6ab362ebc1a5b978b6fc642f51c
0xc3e002ef6aebf9338c7e71e9284c240d6ea6bfaf
0x979148610a48e18475953b81f705197d6f2dac9f
0xe62d5d36884d6d03f6f941a0f3fecf0aa85f802b
0xda1139ba14919d5c08607974ae2490a6d63af057
0x6764162da2d9a27c760074daaa6ffc913f788cd4
0x90c4e1558b67f7452bf01ef7a2c208b8413b8021
0x665a486ea52daa7adecd32ae3eee6ba1075d686b
0x0341c4f8a9a32b2539afa548c875272939e7fd75
0xacaed461c804d2b7ff3582087356e846f1a32416
0x8e430e6ece1c9efc921b592b4efb70261401d577
0x4045b0cbac3079b5119124d2261f06d8e0344d2b
0x99b3c7a3023a94956bb641759cbfe6a6f9d2bef8
0x26126075a7205b4882ab4af83e1aeaf36bfaa7f4
0x6dbb6f607b8bd880ece731c2f207b4f68c5f051a
0x4ccf9b604eb213bf2e8c06dcdbaff6a0c8a8cb9d
0x042456549eb7a3af0d0ffe0242f53e00644c34e0
0xe14632e90bddf50eb4d6976325eb42773b0f111c
0x5fdb62eeb4e8dca2d64866276b1197f8ead269d8
0xeef587fb596746f1006e1ac6fa0d2abbea931937
0x446d8e6113f19bec3bde1b0ddf352739a4c492db
0xc8d99c9a87c66e3f44e12539e68b44aacd173484
0x0943bfa94e20cc54ac6c94e63926890d678828ec
0x15d8a22b38d0cff142fa246d4286b3db73e1a539
0x46862282e63897f9f951665a59174326bd4ce0ea
0xbf31a60a5a40b4c20e8b5a39b5f1593091309dce
0x110df18fb2a2a663bbc5101422686e44a11c33d5
0x82e48558c684d6d27e12e0e0e9ab018e659b8a70
0xb2bf7d050cb1321a2a0fad61bae392834237be70
0x1a1586aaaf48c5da8aac7c54b8f7a0fa09446788
0x7e815d335fd7e4e732115215dcea0e85f1f10c58
0x4420e112a8f92cf8744efdf4f33c8db100201f0d
0x8350035331fe16d80df6718944f7b6040f04d84c
0xfd718d7fd53f85bf7cbbed7484462339ef6aa9df
0x96405385d7e20f0de35c5e50442908d898dfd79d
0xf04c76e2d017404f75cee196eb92a067a7d215d3
0xe09fc9889f06502ad4e9851273469a009c461d14
0x336eb350682d604cd2c54b3d96e9b8fd40c5ae03
0xc7e0c1f10897f6e9eba06c8215313f8624022e1f
0x92402b1009ecab5c7b6fb0da0584c3b5bcf10306
0x5cd7361c5779be57498c1388a10d35380974a46b
0x42e661c76624b0e96d77129e5372032a9cb36d14
0xffef095be1824112d1757dd45ed62350abeae461
0xd6b613cf5bc62d01a37a0b8749742dec4f3a3266
0x899650df74573a763d22c19fcb278dacbd4d84f7
0x81b1b7d95848a99220c7799b261e8655811788a8
0x9add52c096f70a69e0cb6d7e75f6169be07c0363
0xf813c359660b5c453aafa1fe8fd3ae99c7a89eef
0x37c170e25617735fd78d37209e970ab8c76acabb
0xd67ce0b778467eb8d0eeb115cabe0a6c90963a81
0x8803a72403d4f67e8e14dca828a1915f58ff181a
0xc0ecf59a4cf0d099726c596ec1c57a6788fe631f
0x529b6009f3aae9e7e799c73097a617a23f5e1113
0xfd046720d3d396aa615796d7eec77df8e4a9c459
0x819c7d02b2b3ab1fe2c84d12b09e2eed52b7f72b
0x15a2678aff90bddf4b05843995aa1625c1cb3968
0x78b13da46a7ff67f3c1f7ca67b42753cf5bab748
0xe1ef2f25a8a06c5b9c42976f7d07250efa07bc92
0xadef8e502dcd50c76906680ecb16d66db9726d8c
0x7a04b77f8ff4a6c6264291bb48c5c3e1d4281dd6
0x54fa777908c3b0447506fd156a6833256b476f43
0xba9beb18f462d5d1063bcc1266f0861279c14129
0x57301b5ba8003da9f9b92a47bda4bb9b35243780
0x6c07d643fbfc5cc652e849e86edc9ce74085766a
0xdc9bec426303876233ed73905102c73ad354603a
0xcefbaa0cc7ddfeb6971d28ac12857eefad2e6d6c
0x177d1794d0988a0421f8e4655192ffe66ef8254c
0xd1a6765f1a135b2399e97923e408382c0a737dac
0xe6ff83877623276ee6f555ec7266153bdb69fdb8
0xfdd043e8e6b2436401da6542a02913af9085d930
0x246269ac98faf363be9c71c1c404dc7483d2f51f
0x4b831a469c5f60d693c1b0b26fca13a9d3be3266
0x55343140826c93550d916c12c2ea1b7562f460cd
0x37b1c35913a04f085cbd0dda4fcae98c4fe333be
0x0eaddb06b1dcdf8357bac247b3f62ec984bf0e9b
0xcd8c9fe0ad213fce7e7e393869f3672597a4d0e3
0x95697b814799d0c5e583343a6253041d96db7d03
0x2c0c3fd2554785e74a8c1e65d422272974b11045
0x7b53a7da79cd8bc46957cbb6801ef351755a9006
0x52a4fea742180b3120ab97c7f2f52705366de14f
0x35c595c824d954e3b8c63c8b8aa7674a64436f42
0xfc8a0781c189b968b6fc93802dc6bb9232949a1a
0xfd834144dec5a63bb6fe99a68c2b0da4ea057182
0x2e7684e3e76b42236d75e69e68bccb107ef6c386
0xc325a4bfbcacc9f1ff87b0be3e88a5b96e6da221
0x5f0b356a9c75e4cc2ee1b5f24164350e5836ba14
0x1d73d6ff211c209fed030e9c12c8dd8f04489866
0x443a0af9168c85663b1ad244a2095a20ebf1817a
0x97032b34c2aa9b1d9e9d8b9589cd4125ff222cea
0xc7881b30136f23a3273a737bbf44fe53a8a356e0
0x4025c16fc65bd6e81b060523153f8bc99962ca1e
0x8a61c26f99aabed08691d1122a577544d01800d4
0xe4c1234444e895f192849bddaf449b4a781828a8
0xe36d2c4db9a1203d62098f5f78ec65aa7aaf8647
```

# Description
I believe the provided list of addresses is owned by a single entity, based on the following observations:
-	All 254 addresses were created on the same day (2023/6/15).
-	There is a clear consistency in the interaction data among this group of addresses.
-	Consistency in interaction data across other public chains is also evident.

# Detailed Methodology & Walkthrough
The addresses reported were found using the Dune Analytics dashboard. We exported all Layerzero addresses from this dashboard into a CSV file [https://dune.com/fashaking/ultimate-layerzero-checker](https://dune.com/fashaking/ultimate-layerzero-checker). Subsequently, we converted the CSV file into Excel to sort and filter wallets, as illustrated in Figure 1.

## Figure 1
![image](https://github.com/mejiwef/My-L0-Report/blob/main/img/gjmgdzyKKdyAVaNj/1-1.png?raw=true)
![image](https://github.com/mejiwef/My-L0-Report/blob/main/img/gjmgdzyKKdyAVaNj/1-2.png?raw=true)
![image](https://github.com/mejiwef/My-L0-Report/blob/main/img/gjmgdzyKKdyAVaNj/1-3.png?raw=true)
![image](https://github.com/mejiwef/My-L0-Report/blob/main/img/gjmgdzyKKdyAVaNj/1-4.png?raw=true)
![image](https://github.com/mejiwef/My-L0-Report/blob/main/img/gjmgdzyKKdyAVaNj/1-5.png?raw=true)

From this data, the following observations are evident:
-	The initial block usage of all 254 addresses occurred exclusively on the days of 2023/5/14-2023/5/15.
-	The timing of interaction between target and original chains is almost identical.
-	The number of transactions and transaction amounts are nearly identical.

Based on this data, it can be inferred that this group of addresses is likely a set of witch addresses.

To minimize false positives, we also analyzed the interaction gas fees of these addresses on other chains. We utilized the OKLink API (https://github.com/LayerZero-Labs/sybil-report/blob/main/.github/ISSUE_TEMPLATE/sybil-report.md) to retrieve data and examine the transaction gas fees of this group of addresses across various chains, as depicted in Figure 2.

## Figure 2
![image](https://github.com/mejiwef/My-L0-Report/blob/main/img/gjmgdzyKKdyAVaNj/2-1.png?raw=true)
![image](https://github.com/mejiwef/My-L0-Report/blob/main/img/gjmgdzyKKdyAVaNj/2-2.png?raw=true)
![image](https://github.com/mejiwef/My-L0-Report/blob/main/img/gjmgdzyKKdyAVaNj/2-3.png?raw=true)
![image](https://github.com/mejiwef/My-L0-Report/blob/main/img/gjmgdzyKKdyAVaNj/2-4.png?raw=true)
![image](https://github.com/mejiwef/My-L0-Report/blob/main/img/gjmgdzyKKdyAVaNj/2-5.png?raw=true)

-	It can be observed that this group predominantly interacts with BSC, POLYGON, and AVAXC, with minimal or no activity on ETH, OP, ARB, and BASE.
-	Additionally, the gas fees incurred by this group of addresses on BSC, POLYGON, and AVAXC are nearly identical.

In Summary:
The 254 addresses exhibit remarkable consistency in creation time, interaction patterns, and transaction data. Furthermore, the consistency in interaction data across different chains suggests that this group of wallets is likely composed of witch addresses.

# Reward Address (If Eligible)
My Reward Address is 0x268bbb11d6af93c3d5a50bf82c81027a309ed9a9

For any question, you can contact me on:
yangimo217@gmail.com
